Cubical/HITs/Ints/DeltaInt/Base.agda

```diff
@@ -0,0 +1,60 @@
+{-# OPTIONS --cubical --safe #-}
+
+{-
+
+This file defines DiffInt (Cubical.HITs.DiffInt) as HIT.
```

Comment:
I guess it should be `DeltaInt` in both those cases?

Reply:
Nope, I meant the following:

https://github.com/agda/cubical/blob/6ca71118f1275b4b5a6b4c4e37669b6d469fcbad/Cubical/Data/DiffInt.agda#L1-L5

But it's still a typo (`HITs` -> `Data`), I'll fix